propertyestimator/substances.py

```diff
@@ -131,7 +131,7 @@ def __hash__(self):
 
         def __eq__(self, other):
 
-            return (self.smiles == other.self and
+            return (self.smiles == other.smiles and
```

Comment:
Let's start a conversation about where the SMILES come from. For this use case, we might toss in CMILES. It could help with some issues, like formatting alternating C-C single+double bonds into aromatic atoms and handling arbitrary protomers.

Reply:
+1 one for starting to think about this - great catch! Although when we switch over to the OFF toolkit V1.0.0 I think a lot of this functionality will be removed from the estimator, and simply pulled from your toolkit-independent api.